source/image/ImageSet.cpp

```diff
@@ -227,9 +227,9 @@ void ImageSet::Load() noexcept(false)
 	// Warn about a "high-profile" image that will be blurry due to rendering at 50% scale.
 	bool willBlur = (buffer[0].Width() & 1) || (buffer[0].Height() & 1);
 	if(willBlur && (
-			(name.length() > 5 && !name.compare(0, 5, "ship/"))
-			|| (name.length() > 7 && !name.compare(0, 7, "outfit/"))
-			|| (name.length() > 10 && !name.compare(0, 10, "thumbnail/"))
+			(name.length() > 5 && name.starts_with("ship/"))
+			|| (name.length() > 7 && name.starts_with("outfit/"))
+			|| (name.length() > 10 && name.starts_with("thumbnail/"))
```

Comment:
As above, the size check should be already accounted for.
```suggestion
			name.starts_with("ship/")
			|| name.starts_with("outfit/")
			|| name.starts_with("thumbnail/")
```